web/libs/datamanager/src/components/Common/Modal/Modal.scss

```
@@ -0,0 +1,128 @@
+.modal {
+  --transition-duration: 100ms;
+
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  z-index: 10000;
+  display: flex;
+  position: absolute;
+  align-items: center;
+  justify-content: center;
+  background-color: rgba(#000, 0.3);
+  will-change: opacity;
+
+  &__wrapper {
+    width: 100%;
+    max-height: 100%;
+    padding: 40px 0;
+    overflow: auto;
+    box-sizing: border-box;
+    will-change: transform;
+  }
+
+  &__content {
+    width: 400px;
+    min-width: 400px;
+    min-height: 100px;
+    margin: 0 auto;
+    background-color: #fff;
+    border-radius: 5px;
+    box-shadow: 0 10px 30px rgba(#000, 0.3);
```

Flagging as opportunity for css var replacement. Further review of usage needed.
```suggestion
    box-shadow: 0 10px 30px rgba(var(--black), 0.3);
```